Traceback (most recent call last):
  File "C:/Users/Vladislav/PycharmProjects/crashparser/1.py", line 3, in <module>
    raise Exception('x should not exceed 5. The value of x was: {}'.format(x))
Exception: x should not exceed 5. The value of x was: 10

Traceback (most recent call last):
  File "C:/Users/Vladislav/PycharmProjects/crashparser/1.py", line 1, in <module>
    print(open("1.txt", "r"))
IOError: [Errno 2] No such file or directory: '1.txt'

Traceback (most recent call last):
  File "C:/Users/Vladislav/PycharmProjects/crashparser/1.py", line 1, in <module>
    print (1/0)
ZeroDivisionError: integer division or modulo by zero